packages/p2p/src/components/my-ads/edit-ad-form.jsx

```diff
@@ -338,17 +345,18 @@ const EditAdForm = () => {
                 is_open={my_ads_store.is_edit_ad_error_modal_visible}
                 small
                 has_close_icon={false}
-                title={localize('Something’s not right')}
+                title={generateErrorDialogTitle(my_ads_store.error_code)}
             >
                 <Modal.Body>
                     <Text as='p' size='xs' color='prominent'>
-                        {my_ads_store.edit_ad_form_error}
+                        {/* {my_ads_store.edit_ad_form_error} */}
```

Comment:
Could you remove this commented code if it's not required?

Reply:
Will remove it